app/main.py

```diff
@@ -1,6 +1,22 @@
+import requests
+import os
+
+
+URL = "https://api.weatherapi.com/v1/current.json"
+FILTERING = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print("Performing request to Weather API for city Paris...")
+    api_key = os.getenv("API_KEY")
```

Comment:
make it constant, please:)

Reply:
while I don't think that my weather key is something valuable (and I'm currently at the point where I hate env variables), it was in the requirements:
![image](https://user-images.githubusercontent.com/123254785/232593286-cb26631a-0227-4f5e-a31f-9ef173608555.png)

Reply:
It is about following best practices. The same as u've done with `url` and `q`-param